src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/DeleteManagedOrganizationUserAccountCommand.cs

```diff
@@ -36,6 +38,7 @@ public DeleteManagedOrganizationUserAccountCommand(
         _userRepository = userRepository;
         _currentContext = currentContext;
         _hasConfirmedOwnersExceptQuery = hasConfirmedOwnersExceptQuery;
+        _featureService = featureService;
     }
 
     public async Task DeleteUserAsync(Guid organizationId, Guid organizationUserId, Guid? deletingUserId)
```

Comment:
I'd prefere if you create a new version of this method for when the account deprovisioning flag is enabled. So check the flag first thing in this method and redirect to the new one if enabled
```suggestion
    public async Task DeleteUserAsync(Guid organizationId, Guid organizationUserId, Guid? deletingUserId)
    {
        if _featureService.IsEnabled(FeatureFlagKeys.AccountDeprovisioning))
        {
            return await DeleteUserAsync_vNext(organizationId, organizationUserId, deletingUserId);
        }
```

Reply:
But this command is only used by account deprovisioning?

Reply:
Yes, I was confused! I cleared this up with @BTreston and forgot to edit this.